_field-types/mapping-parameters/coerce.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: default
+title: coerce
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 15
+has_children: false
+has_toc: false
+---
+
+# `coerce`
+
+The `coerce` mapping parameter controls how values are converted to the expected data type of a field during indexing. This parameter lets you verify that your data is formatted and indexed properly, following the expected field types. This helps keep your data reliable and improves the accuracy of your search results.
+
+---
+
+## Examples
+
+The following examples show how to use the `coerce` mapping parameter.
+
+#### Indexing a document with `coerce` enabled
+
+```json
+PUT products
+{
+  "mappings": {
+    "properties": {
+      "price": {
+        "type": "integer",
+        "coerce": true
+      }
+    }
+  }
+}
+
+PUT products/_doc/1
+{
+  "name": "Product A",
+  "price": "19.99"
+}
+```
+{% include copy-curl.html %}
+
+In this example, the `price` field is defined as an `integer` type with `coerce` set to `true`. When indexing the document, the string value `19.99` is coerced to the integer `19`.
+
+#### Indexing a document with `coerce` disabled
+
+```json
+PUT orders
+{
+  "mappings": {
+    "properties": {
+      "quantity": {
+        "type": "integer",
+        "coerce": false
+      }
+    }
+  }
+}
+
+PUT orders/_doc/1
+{
+  "item": "Widget",
+  "quantity": "10"
+}
+```
+{% include copy-curl.html %}
+
+In this example, the `quantity` field is defined as an `integer` type with `coerce` set to `false`. When indexing the document, the string value `10` is not coerced, and the document is rejected due to the type mismatch. 
```

Comment:
```suggestion
In this example, the `quantity` field is defined as an `integer` type with `coerce` set to `false`. When indexing the document, the string value `10` is not coerced, and the document is rejected because of the type mismatch. 
```